pyfarm/agent/entrypoints/main.py

```diff
@@ -581,7 +581,7 @@ def start(self):
             logger.debug("Creating %s", config["jobtype_task_logs"])
             try:
                 os.makedirs(config["jobtype_task_logs"])
-            except OSError:
+            except (OSError, WindowsError):
```

Comment:
You're going to want to put this at the top:

```python
try:
    WindowsError
except NameError:  # pragma: no cover
    WindowsError = OSError
```

`WindowError` won't be defined for anything except Windows.